.github/workflows/test.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         run: npm run build
 
       - name: Test/Coverage 🧪
-        run: npm run coverage
+        run: TEST_TIMEOUT_FACTOR=3 npm run coverage
```

Comment:
We can triple the timeout for github actions, but leave our local tests at a reasonable time.